src/classes/EP_Task_UTIL.cls

```diff
@@ -39,6 +39,9 @@
 */
 public class EP_Task_UTIL {
```

Comment:
This class does not have its own test class but is (in)directly tested in the EP_* classes where it is being called. Do we need new or adjusted existing tests in the EP_*_TEST classes to verify new functionality?

Reply:
It would likely need regression flows that specifically create and map custom fields on the Plan Template and Task objects.  That felt out of scope for me, for the amount of time I had to work on this.